xbmc/GUIInfoManager.cpp

```diff
@@ -1741,6 +1741,17 @@ const infomap weather[] =        {{ "isfetched",        WEATHER_IS_FETCHED },
 ///     @return the current language.
 ///     <p>
 ///   }
+///   \table_row3{   <b>`System.Locate(type)`</b>,
+///                  \anchor System_Locale
```

Comment:
This misses revision tags, please do something similar to https://github.com/xbmc/xbmc/blob/master/xbmc/GUIInfoManager.cpp#L149-L151